lib/v2/screens/onboarding/components/pages/onboarding_page_3.dart

```diff
@@ -11,7 +11,7 @@ class ThirdPage extends StatelessWidget {
       topPadding: 50,
       title: "Regenerative Economy",
       subTitle:
-          "Unite with a global movement of organisations and people to regenerate our planet and heal our economy.",
+          "Unite with a global movement of organizations and people to regenerate our planet and heal our economy.",
```

Comment:
I have learned from our original marketing team ages ago that the spelling with 's' is also legitimate. 

But - I much prefer z. S just looks weird to me.

Reply:
History - they decided to spell it with S originally. 

But... I think that's just really strange, and prefer Z.